internal/ignition/ignition.go

```diff
@@ -1381,7 +1397,7 @@ func (ib *ignitionBuilder) prepareStaticNetworkConfigForIgnition(ctx context.Con
 	return filesList, nil
 }
 
-func (ib *ignitionBuilder) FormatSecondDayWorkerIgnitionFile(address string, machineConfigPoolName string) ([]byte, error) {
+func (ib *ignitionBuilder) FormatSecondDayWorkerIgnitionFile(address string, machineConfigPoolName string, host *models.Host) ([]byte, error) {
```

Comment:
I'm not sure what's going on with the diff but the original takes care of HEADERS and CACERT and I don't see it here

https://github.com/openshift/assisted-service/blob/fd010d669fa32b3fa0186e5c6a3e14db4d4861dd/internal/ignition/ignition.go#L1511

Reply:
HEADERS and CACERT are on master, but in 2.4 they were never backported. This is one of the reasons why this PR is effectively a manual work instead of cherry-pick